build.go

```diff
@@ -18,9 +18,15 @@ import (
 )
 
 type BuildOptions struct {
-	Indent int
-	Tabs   bool
-	Header bool
+	Indent        int
+	Tabs          bool
+	Header        bool
+	ExternalBuild []ExtBuild // handle specific directives
```

Comment:
Nit, pluralize the field name as it's a slice?

Reply:
done